app/services/PaypalService.scala

```diff
@@ -289,6 +289,8 @@ class PaypalService(
     idUser.map { id =>
       identityService.updateMarketingPreferences(id, marketingOptInt)
     }
+    if (marketingOptInt)
```

Comment:
we can remove this, plus lines 289 to 291, plus `marketingOptInt` param from this function

Reply:
sorry - I mean, you can get rid of this whole method!